src/build.c

```diff
@@ -158,6 +158,9 @@ widgets;
 static guint build_groups_count[GEANY_GBG_COUNT] = { 3, 4, 2 };
 static guint build_items_count = 9;
 
+/* include the fixed widgets in an array indexed by groups */
+#define GBG_FIXED GEANY_GBG_COUNT
```

Comment:
Would be better to define in the enum in the header, or if you don't want it in the enum at least next to it so all GBG_* declarations are in the same place.

Reply:
I've reverted moving GBG_FIXED. It was protected by GEANY_PRIVATE unlike the GEANY_GBG_* enums.